app/controllers/batches_controller.rb

```diff
@@ -49,10 +51,14 @@ def create
       recipe.reload
       recipe.schedule.name.gsub!('Copy', "##{recipe.id}")
       recipe.name.gsub!('Copy', "##{recipe.id}")
+
       @batch.recipe = recipe
+        
+      if @batch.save(validate: false)
+        Recipe.update(recipe.id, :batch_id => @batch.id)
+        #why was this here? it undid all the recipe manipulation so batch.recipe was nil again.
+        #@batch.reload
```

Comment:
Ok, it seemed to be breaking something at some point, but perhaps I fixed a different bug at the same time and didn't realize it.